.travis.yml

```diff
@@ -1,10 +1,13 @@
 language: node_js
 
 node_js:
-  - "8.10"
-  - "lts/*"
+  - "10"
   - "12"
 
+allow_failures:
+  node_js:
+    - "14"
```

Comment:
Almost :)
You should use jobs instead (defined few lines below):

```yaml
language: node_js

jobs:
  allow_failures:
    - node_js: "14"
  include:
    - node_js: "10"
    - node_js: "12"
    - node_js: "14"
    - stage: github page
# etc.
```

Then you can remove the `node_js:` key from line 3

Reply:
Ok thanks a lot!